src/approaches/bilevel_planning_approach.py

```diff
@@ -89,6 +80,25 @@ def _policy(s: State) -> Action:
 
         return _policy
 
+    def _run_sesame_plan(self, task: Task, nsrts: Set[NSRT],
```

Comment:
alternative (preferred??) implementation of what youre trying to do here: make this method have kwargs, this class doesnt use any, but the pg4 run_sesame_plan just calls super() with the extra kwarg of abstract_policy